### ISSUE TITLE
[Design/Content] My VA: Cerner go-live updates for Spokane facilities on 10/24

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#13831]
## Background

The VA has started adding a new health care platform — Cerner — to its ecosystem. This is a portal, like MyHealtheVet, where people can manage their health care information. The VA's goal is to eventually move all health care functionality so it is supported by Cerner, but this roll out will take years. The VA is launching to one facility at a time and in the meantime, VA.gov has to exist in a world where it points people to Cerner or MHV depending on which facility they go to.

The initial Cerner launch for Columbus, OH occurred a couple weeks ago. **The next go-live is for facilities in Spokane, which is happening 10/24.**

**How this affects our team**

In most cases, we are no longer doing active development on My VA. However, the one exception is that we need My VA to account for this Cerner/MHV split. This means that for people who are patients at facilities in Spokane, we need to direct them to Cerner for their prescriptions/messages/appointments and to MHV for any prescriptions/messages/appointments that might be managed at facilities outside Spokane (note: it is possible for veterans to receive care in multiple areas, e.g. think people who live in different places at different times of year).

We have already done this for patients in Columbus, OH. Columbus is only doing appointment scheduling (not prescriptions and messages) through Cerner, and you can see details about that in #12762. We should take a similar approach for Spokane — showing messaging that guides people to Cerner if they are a patient there, and to MHV for everything else.

**Note**: For patients who do not go to Columbus or Spokane facilities, My VA is not changing and will stay just as it is now.

## Tasks

Update My VA to account for patients who need to be directed to Cerner as follows:

- [ ] Content: Provide guidance on approach for how this is being handled elsewhere on the site.
- [ ] Tressa or Cassandra: Provide a mockup of how the My VA dashboard will change for patients at facilities in Spokane.
  - Show prompts in the health care section for prescriptions, messages, and appointments
  - Figure out how to reconcile that for appointments, we may need to direct patients in Columbus, OH, and patients in Spokane, WA to Cerner. **Work with copy to figure out how they are handling this elsewhere on the site**.
- [ ] Content: Provide copy for the My VA prompts.

## Acceptance criteria

- [ ] Updates to My VA are fully ready to hand off to FE for build.

**Comment by Samara-Strauss** [comment]
Placeholder ticket for FE is #13832. Will further groom that ticket once design/content is complete.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @Samara-Strauss @tressaellen @callen2563. Here's how we handled the alerts for the health tools. 

Here's a screenshot of one of the alerts to show the component. All of the alerts follow the same pattern.

![image](https://user-images.githubusercontent.com/62957278/94217197-66769200-feaf-11ea-916e-d97608681bb1.png)

And here's content for secure messaging, prescriptions, and appointments. We also have this component for lab + test results and get medical records, but I don't think you need those, right? If you do, just let me know. And feel  free to reach out with any questions. Thanks!

## Send or receive a secure message

_[alert]_

### Your VA health care team may be using our new My VA Health portal

#### Our records show you’re registered at:

**Mann-Grandstaff VA medical center** (Using VA My Health)

Please choose a health management portal below, depending on your provider’s facility. You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the  same account you used to sign in on VA.gov. 

#### Send a secure message to a provider at:

**Mann-Grandstaff VA Medical Center**

_[button]_ Go to My VA Health

**Another VA health facility**

_[button]_ Go to My HealtheVet

_[end alert]_

## Refill and track prescriptions

_[alert]_

### Your VA health care team may be using our new My VA Health portal

#### Our records show you’re registered at:

Mann-Grandstaff VA medical center (Using VA My Health)
Please choose a health management portal below, depending on your facility. You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the  same account you used to sign in on VA.gov. 

#### Refill prescriptions from:

**Mann-Grandstaff VA Medical Center** 

_[button]_ Go to My VA Health

Another VA health facility

_[button]_ Go to My HealtheVet

_[end alert]_

## View, schedule, or cancel an appointment

_[alert]_

### Your VA health care team may be using our new My VA Health portal

#### Our records show you’re registered at:

**Mann-Grandstaff VA Medical Center** (Using VA My Health)

Please choose a health management portal below, depending on the facility for your appointment. You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the  same account you used to sign in on VA.gov. 

#### Manage appointments at:

**Mann-Grandstaff VA Medical Center**

_[button]_ Go to My VA Health

**Another VA health facility**

_[button]_ Go to the VA appointments tool

_[end alert]_

**Comment by Samara-Strauss** [comment]
@DanielleThierryUSDSVA thank you for this. How would we handle things if someone was a patient at Mann-Grandstaff and Chalmers P. Wylie? I realize that's very unlikely, but I wanted to see if you all had started to think about the use case where we're handling a bunch of facilities for Cerner and someone might be a patient at more than one.

Also, is it safe to assume "Mann-Grandstaff VA medical center (Using VA My Health)" under the "Our records show you’re registered at:" heading is supposed to be bolded for Refill Prescriptions like it is for the other two sections? It's not, but assuming that was a typo and the formatting should be the same as the other two sections.

Thank you so much for your help!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @Samara-Strauss. Correct on the bold - I've fixed above, thanks! 

For 2 facilities, I've added the treatment that I believe we're going with and asked our PW engineers to confirm. Will update here with final confirmation. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
@Samara-Strauss Confirmed on treatment for 2 facilities. Thanks!

**Comment by Samara-Strauss** [comment]
Great, thanks, @DanielleSoCompany! This is super helpful.

**Comment by tressaellen** [comment]
Hey @Samara-Strauss and @DanielleSoCompany here are the mocks with the updated content. Let me know if these need any other updates at all.

Also, I took a look on staging, and it doesn't look like there is any other information included following the alert (i.e. a list of prescriptions etc) - I recommend we just stick to that pattern.

![Screen Shot 2020-09-29 at 1.30.10 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/4d2edc9c-1405-4824-addf-01159e4d7ad4)![Screen Shot 2020-09-29 at 1.30.15 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/0f4401d1-34bc-4932-aa64-eba1d0724e16)![Screen Shot 2020-09-29 at 1.30.21 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/8e2038d4-2b2b-41d8-a86e-7d5d4612ab0a)![Screen Shot 2020-09-29 at 1.30.36 PM.png](https://images.zenhubusercontent.com/5dfa65bbbd078d1699e52294/e68d527a-1faf-44db-bd31-9a6a8c0153d7)

**Comment by Samara-Strauss** [comment]
Thank you for this, @tressaellen!

**Comment by Samara-Strauss** [comment]
FE ticket #13832 has been updated. Closing this.
